src/main/java/hudson/plugins/copyartifact/TriggeredBuildSelector.java

```diff
@@ -93,16 +95,18 @@ public boolean isForGlobalSetting() {
     };
     private Boolean fallbackToLastSuccessful;
     private final UpstreamFilterStrategy upstreamFilterStrategy;
+    private Boolean allowUpstreamDependencies;
 
     @DataBoundConstructor
-    public TriggeredBuildSelector(boolean fallbackToLastSuccessful, UpstreamFilterStrategy upstreamFilterStrategy) {
+    public TriggeredBuildSelector(boolean fallbackToLastSuccessful, UpstreamFilterStrategy upstreamFilterStrategy, boolean allowUpstreamDependencies) {
```

Comment:
Please add an overloaded constructor to keep comparibility (like `TriggerredBuildSelector(boolean)`).